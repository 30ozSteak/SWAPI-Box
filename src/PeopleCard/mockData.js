const people = {
    "count": 87, 
    "next": "https://swapi.co/api/people/?page=2", 
    "previous": null, 
    "results": [
        {
            "name": "Luke Skywalker", 
            "height": "172", 
            "mass": "77", 
            "hair_color": "blond", 
            "skin_color": "fair", 
            "eye_color": "blue", 
            "birth_year": "19BBY", 
            "gender": "male", 
            "homeworld": "https://swapi.co/api/planets/1/", 
            "films": [
                "https://swapi.co/api/films/2/", 
                "https://swapi.co/api/films/6/", 
                "https://swapi.co/api/films/3/", 
                "https://swapi.co/api/films/1/", 
                "https://swapi.co/api/films/7/"
            ], 
            "species": [
                "https://swapi.co/api/species/1/"
            ], 
            "vehicles": [
                "https://swapi.co/api/vehicles/14/", 
                "https://swapi.co/api/vehicles/30/"
            ], 
            "starships": [
                "https://swapi.co/api/starships/12/", 
                "https://swapi.co/api/starships/22/"
            ], 
            "created": "2014-12-09T13:50:51.644000Z", 
            "edited": "2014-12-20T21:17:56.891000Z", 
            "url": "https://swapi.co/api/people/1/"
        }, 
        {
            "name": "C-3PO", 
            "height": "167", 
            "mass": "75", 
            "hair_color": "n/a", 
            "skin_color": "gold", 
            "eye_color": "yellow", 
            "birth_year": "112BBY", 
            "gender": "n/a", 
            "homeworld": "https://swapi.co/api/planets/1/", 
            "films": [
                "https://swapi.co/api/films/2/", 
                "https://swapi.co/api/films/5/", 
                "https://swapi.co/api/films/4/", 
                "https://swapi.co/api/films/6/", 
                "https://swapi.co/api/films/3/", 
                "https://swapi.co/api/films/1/"
            ], 
            "species": [
                "https://swapi.co/api/species/2/"
            ], 
            "vehicles": [], 
            "starships": [], 
            "created": "2014-12-10T15:10:51.357000Z", 
            "edited": "2014-12-20T21:17:50.309000Z", 
            "url": "https://swapi.co/api/people/2/"
        }
    ]
}

const planets = [
        [{
            "name": "Alderaan", 
            "rotation_period": "24", 
            "orbital_period": "364", 
            "diameter": "12500", 
            "climate": "temperate", 
            "gravity": "1 standard", 
            "terrain": "grasslands, mountains", 
            "surface_water": "40", 
            "population": "2000000000", 
            "residents": [
                "https://swapi.co/api/people/5/", 
                "https://swapi.co/api/people/68/", 
                "https://swapi.co/api/people/81/"
            ], 
            "films": [
                "https://swapi.co/api/films/6/", 
                "https://swapi.co/api/films/1/"
            ], 
            "created": "2014-12-10T11:35:48.479000Z", 
            "edited": "2014-12-20T20:58:18.420000Z", 
            "url": "https://swapi.co/api/planets/2/"
        }, 
        {
            "name": "Yavin IV", 
            "rotation_period": "24", 
            "orbital_period": "4818", 
            "diameter": "10200", 
            "climate": "temperate, tropical", 
            "gravity": "1 standard", 
            "terrain": "jungle, rainforests", 
            "surface_water": "8", 
            "population": "1000", 
            "residents": [], 
            "films": [
                "https://swapi.co/api/films/1/"
            ], 
            "created": "2014-12-10T11:37:19.144000Z", 
            "edited": "2014-12-20T20:58:18.421000Z", 
            "url": "https://swapi.co/api/planets/3/"
        }]
    ]

export {
    people
}

export {
    planets
}