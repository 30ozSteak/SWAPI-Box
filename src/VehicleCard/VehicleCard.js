import React from 'react';
import "../PeopleCard/PeopleCard.css";
import PropTypes from "prop-types";
import Images from "../Images.json";

const VehicleCard = ({ vehicle, handleFavorites }) => {
  let vehicleArray = vehicle.results;

  const vehicleStats = vehicleArray.map(data => {
    return (
      <div key={data.name} className="card-literal">
        <section className="image">
          <img
            className="image-literal"
            src={Images[data.name]}
            alt={data.name}
          />
        </section>
        <div className="fave" onClick={() => handleFavorites(data)} />
        <section className="text-field">
          <h2>
            {" "}
            {data.name}
          </h2>

          <h4>
            {" "}
            Model: {data.model}
          </h4>
          <h4>
            {" "}
            Class: {data.vehicle_class}
          </h4>
          <h4>
            {" "}
            Capacity: {data.passengers}
          </h4>
        </section>
      </div>
    );
  });

  return <div>{vehicleStats}</div>;
};

VehicleCard.propTypes = {
  vehicle: PropTypes.array,
  handleFavorites: PropTypes.func
};

export default VehicleCard;
