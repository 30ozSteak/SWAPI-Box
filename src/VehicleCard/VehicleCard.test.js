import React from "react";
import VehicleCard from "./VehicleCard";
import { vehicles } from './mockData'
import { shallow } from "enzyme";

describe("VehicleCard", () => {

  it("should match the screenshot", () => {

    const wrapper = shallow(<VehicleCard vehicle={vehicles} />);

    expect(wrapper).toMatchSnapshot();
  });
})
