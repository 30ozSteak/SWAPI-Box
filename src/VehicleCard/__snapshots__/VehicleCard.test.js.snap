// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VehicleCard should match the screenshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <VehicleCard
    vehicle={
      Object {
        "count": 39,
        "next": "https://swapi.co/api/vehicles/?page=2",
        "previous": null,
        "results": Array [
          Object {
            "cargo_capacity": "50000",
            "consumables": "2 months",
            "cost_in_credits": "150000",
            "created": "2014-12-10T15:36:25.724000Z",
            "crew": "46",
            "edited": "2014-12-22T18:21:15.523587Z",
            "films": Array [
              "https://swapi.co/api/films/5/",
              "https://swapi.co/api/films/1/",
            ],
            "length": "36.8",
            "manufacturer": "Corellia Mining Corporation",
            "max_atmosphering_speed": "30",
            "model": "Digger Crawler",
            "name": "Sand Crawler",
            "passengers": "30",
            "pilots": Array [],
            "url": "https://swapi.co/api/vehicles/4/",
            "vehicle_class": "wheeled",
          },
          Object {
            "cargo_capacity": "50",
            "consumables": "0",
            "cost_in_credits": "14500",
            "created": "2014-12-10T16:01:52.434000Z",
            "crew": "1",
            "edited": "2014-12-22T18:21:15.552614Z",
            "films": Array [
              "https://swapi.co/api/films/1/",
            ],
            "length": "10.4",
            "manufacturer": "Incom Corporation",
            "max_atmosphering_speed": "1200",
            "model": "T-16 skyhopper",
            "name": "T-16 skyhopper",
            "passengers": "1",
            "pilots": Array [],
            "url": "https://swapi.co/api/vehicles/6/",
            "vehicle_class": "repulsorcraft",
          },
        ],
      }
    }
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <div
          className="card-literal"
        >
          <section
            className="image"
          >
            <img
              alt="Sand Crawler"
              className="image-literal"
              src="../vehicleimages/sandcrawler.png"
            />
          </section>
          <div
            className="fave"
            onClick={[Function]}
          />
          <section
            className="text-field"
          >
            <h2>
               
              Sand Crawler
            </h2>
            <h4>
               
              Model: 
              Digger Crawler
            </h4>
            <h4>
               
              Class: 
              wheeled
            </h4>
            <h4>
               
              Capacity: 
              30
            </h4>
          </section>
        </div>,
        <div
          className="card-literal"
        >
          <section
            className="image"
          >
            <img
              alt="T-16 skyhopper"
              className="image-literal"
              src="../vehicleimages/t16skyhopper.png"
            />
          </section>
          <div
            className="fave"
            onClick={[Function]}
          />
          <section
            className="text-field"
          >
            <h2>
               
              T-16 skyhopper
            </h2>
            <h4>
               
              Model: 
              T-16 skyhopper
            </h4>
            <h4>
               
              Class: 
              repulsorcraft
            </h4>
            <h4>
               
              Capacity: 
              1
            </h4>
          </section>
        </div>,
      ],
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": "Sand Crawler",
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <section
              className="image"
            >
              <img
                alt="Sand Crawler"
                className="image-literal"
                src="../vehicleimages/sandcrawler.png"
              />
            </section>,
            <div
              className="fave"
              onClick={[Function]}
            />,
            <section
              className="text-field"
            >
              <h2>
                 
                Sand Crawler
              </h2>
              <h4>
                 
                Model: 
                Digger Crawler
              </h4>
              <h4>
                 
                Class: 
                wheeled
              </h4>
              <h4>
                 
                Capacity: 
                30
              </h4>
            </section>,
          ],
          "className": "card-literal",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="Sand Crawler"
                className="image-literal"
                src="../vehicleimages/sandcrawler.png"
              />,
              "className": "image",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "Sand Crawler",
                "className": "image-literal",
                "src": "../vehicleimages/sandcrawler.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fave",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                   
                  Sand Crawler
                </h2>,
                <h4>
                   
                  Model: 
                  Digger Crawler
                </h4>,
                <h4>
                   
                  Class: 
                  wheeled
                </h4>,
                <h4>
                   
                  Capacity: 
                  30
                </h4>,
              ],
              "className": "text-field",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Sand Crawler",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Sand Crawler",
                ],
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Model: ",
                    "Digger Crawler",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Model: ",
                  "Digger Crawler",
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Class: ",
                    "wheeled",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Class: ",
                  "wheeled",
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Capacity: ",
                    "30",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Capacity: ",
                  "30",
                ],
                "type": "h4",
              },
            ],
            "type": "section",
          },
        ],
        "type": "div",
      },
      Object {
        "instance": null,
        "key": "T-16 skyhopper",
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <section
              className="image"
            >
              <img
                alt="T-16 skyhopper"
                className="image-literal"
                src="../vehicleimages/t16skyhopper.png"
              />
            </section>,
            <div
              className="fave"
              onClick={[Function]}
            />,
            <section
              className="text-field"
            >
              <h2>
                 
                T-16 skyhopper
              </h2>
              <h4>
                 
                Model: 
                T-16 skyhopper
              </h4>
              <h4>
                 
                Class: 
                repulsorcraft
              </h4>
              <h4>
                 
                Capacity: 
                1
              </h4>
            </section>,
          ],
          "className": "card-literal",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": <img
                alt="T-16 skyhopper"
                className="image-literal"
                src="../vehicleimages/t16skyhopper.png"
              />,
              "className": "image",
            },
            "ref": null,
            "rendered": Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "alt": "T-16 skyhopper",
                "className": "image-literal",
                "src": "../vehicleimages/t16skyhopper.png",
              },
              "ref": null,
              "rendered": null,
              "type": "img",
            },
            "type": "section",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "className": "fave",
              "onClick": [Function],
            },
            "ref": null,
            "rendered": null,
            "type": "div",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "children": Array [
                <h2>
                   
                  T-16 skyhopper
                </h2>,
                <h4>
                   
                  Model: 
                  T-16 skyhopper
                </h4>,
                <h4>
                   
                  Class: 
                  repulsorcraft
                </h4>,
                <h4>
                   
                  Capacity: 
                  1
                </h4>,
              ],
              "className": "text-field",
            },
            "ref": null,
            "rendered": Array [
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "T-16 skyhopper",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "T-16 skyhopper",
                ],
                "type": "h2",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Model: ",
                    "T-16 skyhopper",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Model: ",
                  "T-16 skyhopper",
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Class: ",
                    "repulsorcraft",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Class: ",
                  "repulsorcraft",
                ],
                "type": "h4",
              },
              Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "children": Array [
                    " ",
                    "Capacity: ",
                    "1",
                  ],
                },
                "ref": null,
                "rendered": Array [
                  " ",
                  "Capacity: ",
                  "1",
                ],
                "type": "h4",
              },
            ],
            "type": "section",
          },
        ],
        "type": "div",
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <div
            className="card-literal"
          >
            <section
              className="image"
            >
              <img
                alt="Sand Crawler"
                className="image-literal"
                src="../vehicleimages/sandcrawler.png"
              />
            </section>
            <div
              className="fave"
              onClick={[Function]}
            />
            <section
              className="text-field"
            >
              <h2>
                 
                Sand Crawler
              </h2>
              <h4>
                 
                Model: 
                Digger Crawler
              </h4>
              <h4>
                 
                Class: 
                wheeled
              </h4>
              <h4>
                 
                Capacity: 
                30
              </h4>
            </section>
          </div>,
          <div
            className="card-literal"
          >
            <section
              className="image"
            >
              <img
                alt="T-16 skyhopper"
                className="image-literal"
                src="../vehicleimages/t16skyhopper.png"
              />
            </section>
            <div
              className="fave"
              onClick={[Function]}
            />
            <section
              className="text-field"
            >
              <h2>
                 
                T-16 skyhopper
              </h2>
              <h4>
                 
                Model: 
                T-16 skyhopper
              </h4>
              <h4>
                 
                Class: 
                repulsorcraft
              </h4>
              <h4>
                 
                Capacity: 
                1
              </h4>
            </section>
          </div>,
        ],
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": "Sand Crawler",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <section
                className="image"
              >
                <img
                  alt="Sand Crawler"
                  className="image-literal"
                  src="../vehicleimages/sandcrawler.png"
                />
              </section>,
              <div
                className="fave"
                onClick={[Function]}
              />,
              <section
                className="text-field"
              >
                <h2>
                   
                  Sand Crawler
                </h2>
                <h4>
                   
                  Model: 
                  Digger Crawler
                </h4>
                <h4>
                   
                  Class: 
                  wheeled
                </h4>
                <h4>
                   
                  Capacity: 
                  30
                </h4>
              </section>,
            ],
            "className": "card-literal",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="Sand Crawler"
                  className="image-literal"
                  src="../vehicleimages/sandcrawler.png"
                />,
                "className": "image",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "Sand Crawler",
                  "className": "image-literal",
                  "src": "../vehicleimages/sandcrawler.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "section",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fave",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                     
                    Sand Crawler
                  </h2>,
                  <h4>
                     
                    Model: 
                    Digger Crawler
                  </h4>,
                  <h4>
                     
                    Class: 
                    wheeled
                  </h4>,
                  <h4>
                     
                    Capacity: 
                    30
                  </h4>,
                ],
                "className": "text-field",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Sand Crawler",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Sand Crawler",
                  ],
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Model: ",
                      "Digger Crawler",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Model: ",
                    "Digger Crawler",
                  ],
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Class: ",
                      "wheeled",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Class: ",
                    "wheeled",
                  ],
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Capacity: ",
                      "30",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Capacity: ",
                    "30",
                  ],
                  "type": "h4",
                },
              ],
              "type": "section",
            },
          ],
          "type": "div",
        },
        Object {
          "instance": null,
          "key": "T-16 skyhopper",
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <section
                className="image"
              >
                <img
                  alt="T-16 skyhopper"
                  className="image-literal"
                  src="../vehicleimages/t16skyhopper.png"
                />
              </section>,
              <div
                className="fave"
                onClick={[Function]}
              />,
              <section
                className="text-field"
              >
                <h2>
                   
                  T-16 skyhopper
                </h2>
                <h4>
                   
                  Model: 
                  T-16 skyhopper
                </h4>
                <h4>
                   
                  Class: 
                  repulsorcraft
                </h4>
                <h4>
                   
                  Capacity: 
                  1
                </h4>
              </section>,
            ],
            "className": "card-literal",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": <img
                  alt="T-16 skyhopper"
                  className="image-literal"
                  src="../vehicleimages/t16skyhopper.png"
                />,
                "className": "image",
              },
              "ref": null,
              "rendered": Object {
                "instance": null,
                "key": undefined,
                "nodeType": "host",
                "props": Object {
                  "alt": "T-16 skyhopper",
                  "className": "image-literal",
                  "src": "../vehicleimages/t16skyhopper.png",
                },
                "ref": null,
                "rendered": null,
                "type": "img",
              },
              "type": "section",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "className": "fave",
                "onClick": [Function],
              },
              "ref": null,
              "rendered": null,
              "type": "div",
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "host",
              "props": Object {
                "children": Array [
                  <h2>
                     
                    T-16 skyhopper
                  </h2>,
                  <h4>
                     
                    Model: 
                    T-16 skyhopper
                  </h4>,
                  <h4>
                     
                    Class: 
                    repulsorcraft
                  </h4>,
                  <h4>
                     
                    Capacity: 
                    1
                  </h4>,
                ],
                "className": "text-field",
              },
              "ref": null,
              "rendered": Array [
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "T-16 skyhopper",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "T-16 skyhopper",
                  ],
                  "type": "h2",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Model: ",
                      "T-16 skyhopper",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Model: ",
                    "T-16 skyhopper",
                  ],
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Class: ",
                      "repulsorcraft",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Class: ",
                    "repulsorcraft",
                  ],
                  "type": "h4",
                },
                Object {
                  "instance": null,
                  "key": undefined,
                  "nodeType": "host",
                  "props": Object {
                    "children": Array [
                      " ",
                      "Capacity: ",
                      "1",
                    ],
                  },
                  "ref": null,
                  "rendered": Array [
                    " ",
                    "Capacity: ",
                    "1",
                  ],
                  "type": "h4",
                },
              ],
              "type": "section",
            },
          ],
          "type": "div",
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
